_notifications-plugin/api.md

```diff
@@ -0,0 +1,401 @@
+
+Lists all configurations.
+
+#### Sample Request
+
+```json
+GET _plugins/_notifications/configs
+```
+
+#### Sample Response
+
+```json
+{
+  "start_index" : 0,
+  "total_hits" : 2,
+  "total_hit_relation" : "eq",
+  "config_list" : [
+    {
+      "config_id" : "sample-id",
+      "last_updated_time_ms" : 1652760532774,
+      "created_time_ms" : 1652760532774,
+      "config" : {
+        "name" : "Sample Slack Channel",
+        "description" : "This is a Slack channel",
+        "config_type" : "slack",
+        "is_enabled" : true,
+        "slack" : {
+          "url" : "https://sample-slack-webhook"
+        }
+      }
+    },
+    {
+      "config_id" : "sample-id2",
+      "last_updated_time_ms" : 1652760735380,
+      "created_time_ms" : 1652760735380,
+      "config" : {
+        "name" : "Test chime channel",
+        "description" : "A test chime channel",
+        "config_type" : "chime",
+        "is_enabled" : true,
+        "chime" : {
+          "url" : "https://sample-chime-webhook"
+        }
+      }
+    }
+  ]
+}
+```
+
+You can include query parameters in your request path to filter the notification channels this request returns. All parameters are optional.
+
+Parameter	| Description
+:--- | :---
+config_id | Specifies the channel identifier.
+config_id_list | Specifies a comma-separated list of channel IDs.
+from_index | The starting index to search from.
+max_items | The maximum amount of items to return in your request.
+sort_order | Specifies the direction to sort results in. Valid options are `asc` and `desc`.
+sort_field | Field to sort results with.
+last_updated_time_ms | The unix time in milliseconds of when the channel was last updated.
+created_time_ms | The unix time in milliseconds of when the channel was created.
+is_enabled | Indicates whether the channel is enabled.
+config_type | The channel type. Valid options are `sns`, `slack`, `chime`, `webhook`, `smtp_account`, `ses_account`, `email_group`, `email`.
+name | The channel's name.
+description	| The channel's description.
+email.email_account_id | The sender emails the channel uses.
+email.email_group_id_list | The email groups the channel uses.
+email.recipient_list | The channel's recipient list.
+email_group.recipient_list | The channel's list of email recipient groups.
+smtp_account.method | The email encryption method.
+slack.url	| The Slack channel's URL.
+chime.url	| The Amazon Chime connection's URL.
+webhook.url	| The webhook's URL.
+smtp_account.host	| The domain of the smtp account.
+smtp_account.from_address	| The email account's sender address.
+smtp_account.method | The smtp account's encryption method.
+sns.topic_arn	| The Amazon SNS topic's ARN.
+sns.role_arn | The Amazon SNS topic's role ARN.
+ses_account.region | The Amazon SES account's region.
+ses_account.role_arn | The Amazon SES account's role ARN.
+ses_account.from_address | The Amazon SES account's sender email address.
+
+## Create channel configuration
```

Comment:
```suggestion
## Create a new notifications channel
```

Reply:
I'm leaving it as `Create channel configuration`, because I think "create" and "new" are redundant together.